File: code-examples/java/users/fb_auth.java
HttpResponse<String> response = Unirest.post("http://api.marketcloud.it/v0/users/authenticate/facebook")
  .header("authorization", "YourPublicKey:YourToken")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .body("{\"access_token\" : \"user-access-token\" ,\"user_id\" : \"the-user-id\"}")
  .asString();

File: code-examples/csharp/users/fb_auth.cs
var client = new RestClient("http://api.marketcloud.it/v0/users/authenticate/facebook");
var request = new RestRequest(Method.POST);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddParameter("application/json", "{\"access_token\" : \"user-access-token\" ,\"user_id\" : \"the-user-id\"}", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);